developer_manual/core/unit-testing.rst

```diff
@@ -12,7 +12,13 @@ Getting PHPUnit
 ownCloud >= 10.0
 ^^^^^^^^^^^^^^^^
 
-If you are using ownCloud 10.0 or higher, running ``make test-php`` or ``make`` in your terminal from the root directory will install a local version of PHPUnit inside of ``lib/composer`` and run the test suite.
+If you are using ownCloud 10.0 or higher, running ``make`` in your terminal from the ``webroot`` directory will prepare everything for testing. 
+This will install beside necessary dependencies, a local version of PHPUnit at ``<webroot>/lib/composer/phpunit/phpunit``.
```

Comment:
When I installed phpunit manually, it always took that one instead the provided one and testing did not work (needs investigation). Using teh provided phpunit worked - therefore the path.